src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -132,25 +146,29 @@ function ExportAnnotations({
           <label htmlFor="export-user" className="block font-medium">
             Select which user{"'"}s annotations to export:
           </label>
-          <Select
-            id="export-user"
-            onChange={e =>
-              setSelectedUser((e.target as HTMLSelectElement).value || null)
-            }
+          <SelectNext
+            value={selectedUser}
+            onChange={setSelectedUser}
+            label={<UserAnnotationsContent userAnnotations={selectedUser} />}
           >
-            <option value="" selected={!selectedUser}>
-              All annotations ({exportableAnnotations.length})
-            </option>
+            <SelectNext.Option value={allAnnotationsOption}>
+              {() => (
+                <UserAnnotationsContent
+                  userAnnotations={allAnnotationsOption}
+                />
+              )}
+            </SelectNext.Option>
             {userList.map(userInfo => (
-              <option
-                key={userInfo.userid}
-                value={userInfo.userid}
-                selected={userInfo.userid === selectedUser}
-              >
-                {userInfo.displayName} ({userInfo.annotations.length})
-              </option>
+              <SelectNext.Option key={userInfo.userid} value={userInfo}>
+                {({ selected }) => (
+                  <UserAnnotationsContent
+                    userAnnotations={userInfo}
+                    classes={classnames({ 'font-semibold': selected })}
```

Comment:
Maybe `UserAnnotationsContent` (now `UserAnnotationsListItem`) should accept a boolean `selected` prop, instead of `classes`. That would reduce duplication.

On the other hand, we can just stick with the default if you think that's better. I don't have a strong opinion here.

![image](https://github.com/hypothesis/client/assets/2719332/7d426299-42fc-45a1-b2fe-ea267db88fb9)

 vs

![image](https://github.com/hypothesis/client/assets/2719332/22567435-6dca-4838-a0fc-5e8e1cdc52c5)